easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0-foss-2020b.eb

```
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.0.0'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyYAML', '5.3.1'),
+    ('TensorFlow', '2.5.0'),
+    ('Biopython', '1.78'),
+    ('jax', '0.2.19'),  # also provides absl-py
+]
+
+use_pip = True
+
+exts_list = [
```

```suggestion
exts_list = [
    ('pdbfixer', '1.7', {
        'source_urls': ['https://github.com/openmm/pdbfixer/archive/refs/tags/'],
        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
        'checksums': ['a0bef3c52a7bbe69a6aea5333f51f3e7d158339be5829aed19b0344bd66d4eea'],
    }),
```

This is used in the Amber Relaxation stage to clean the protein.
```
$ python -c "from alphafold.relax import relax"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/rds/projects/2017/branfosj-rse/alphafold/alphafold/alphafold/relax/relax.py", line 18, in <module>
    from alphafold.relax import amber_minimize
  File "/rds/projects/2017/branfosj-rse/alphafold/alphafold/alphafold/relax/amber_minimize.py", line 25, in <module>
    from alphafold.relax import cleanup
  File "/rds/projects/2017/branfosj-rse/alphafold/alphafold/alphafold/relax/cleanup.py", line 22, in <module>
    import pdbfixer
ModuleNotFoundError: No module named 'pdbfixer'
```

---

@branfosj I've included this as a dependency, since it looks like it could be useful standalone too.